Add GitHub PR Template to encourage linking issues (#10585)
MicrosoftDocs/windows-itpro-docs: .github/pull_request_template.md

<!-- 

> Fill out the information below to help us review this pull request. 

Reviewer: ```suggestion
Fill out the following information to help us review this pull request. 
```

You can delete these comments once you are done.
-->
<!-- 
## Description

If your changes are extensive:
- Uncomment this heading and provide a brief description here.
- List more detailed changes below under the changes heading.
-->

## Why

<!--

> - Briefly describe **why** you made this PR.
> - If this PR relates to an issue, provide the issue number or link.
> - If this PR closes an issue, use a keyword (`Closes #123`).
>   - Using a keyword will ensure the issue is automatically closed once this PR is merged.

Reviewer: ```suggestion
- Briefly describe _why_ you made this pull request.
- If this pull request relates to an issue, provide the issue number or link.
- If this pull request closes an issue, use a keyword (`Closes #123`).
  - Using a keyword will ensure the issue is automatically closed once this pull request is merged.
```

  - For more information, see [Linking a pull request to an issue using a keyword](https://docs.github.com/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
-->

- Closes #[Issue Number]

## Changes

<!--

> - Briefly describe or list **what** this PR changes.

Reviewer: ```suggestion
- Briefly describe or list _what_ this PR changes.
```
